css/styles.css

```diff
@@ -140,3 +141,30 @@ footer {
   border: solid 2px red;
   border-radius: .75em;
 }
+
+@media screen and (max-width: 992px) {
+  h1 {
+    font-size: 6em;
+    margin: 60px 0 40px;
+  }
+  
+  input {
```

Comment:
The inputs still seem too big on mobile devices. Also, the margin between them is a little small.

Reply:
changed that in my new pr. If it still doesnt match then please say how you would like to have it